.circleci/config.yml

```diff
@@ -0,0 +1,73 @@
+version: 2.1
+
+orbs:
+  ci: softsense/ci-tools@0.1.42
+
+jobs:
+  test:
+    docker:
+      - image: circleci/golang:1.12 # Primary container image where all steps run.
+        environment:
+          TZ: UTC
+    environment:
+      # Test result directory. go-short is the name of the test result
+      TEST_RESULTS: /tmp/test-results/go-short
+    steps:
+      - checkout
+      - ci/setup_ssh
```

Comment:
`ci/setup_ssh` is only needed if the steps are required to access private repos. This repo won't do that, and shouldn't, since it's a public repo.

Reply:
Oh right. We could perhaps make this repo private since it's for internal use, but I suppose there isn't really any point. Fixed in 69e957436f93bb19525d69fb20f1acf3a37062d7.